### PR TITLE
Add basic Issue and PR Templates 

### LINKED ISSUE #390: Issue and/or PR Templates
Something I think would be useful for people users new to contributing to open source would be templates for creating issues and maybe PRs. I don't think it needs to be anything extravagant, but something to just let them know what they should put in  when submitting an issue and possibly add some auto tagging.

Initial idea I am thinking is:
* Bug Report
  * What is the bug
  * What device (Phone, tablet, desktop, etc.)
  * Steps to reproduce
  * Screenshots

* Feature Request
  * Describe the feature
  * Is it related to a problem?
  * Did you see this feature somewhere else?

* Bug Fix PR
  * Issue(s) fixed
  * What was done to fix it?

Looking for suggestions of templates or things in templates as well

### PR DESCRIPTION
**Pull request by CGBassPlayer**
Fixes #390

Adds issue templates to help users create issues. Currently the templates are
* Issue (Forms)
  * Bug found on the website
  * Feature Request
  * Issue with development environment
  * Issue with episode information (dead links, typos, etc)
* Pull Request (Markdown)
  * Basic Bug Fix

### CODE REVIEW

**Comment by gerbrent** [comment]
THIS IS SO GREAT - thanks @CGBassPlayer . 
Will have a look in a day or two.. (am travelling to the studio!!)

File: .github/ISSUE_TEMPLATE/bug_shownotes.yml
Comment (elreydetoda) [inline .github/ISSUE_TEMPLATE/bug_shownotes.yml:12]: I like this reminder 😁, but is it possible to link to the issues on the word bug? Just so people know what we mean by that if they don't use github very much.

Also, I think the `!` might get interpreted the wrong way since the message will already be bolded with the header text. Do you think a period would be enough? (If we change it here, then we should change it for the other templates as well (just the areas reminding them to check for previous issues)) 

I know these are small detail, but I'm just trying to think of all the ways it could be mis-understood/interpreted 😅

File: .github/ISSUE_TEMPLATE/bug_shownotes.yml
Comment (elreydetoda) [inline .github/ISSUE_TEMPLATE/bug_shownotes.yml:38]: I like that you give examples for this 😁

File: .github/ISSUE_TEMPLATE/bug_shownotes.yml
Comment (elreydetoda) [inline .github/ISSUE_TEMPLATE/bug_shownotes.yml:47]: Ya, we should probably make one of these, I think this is a pretty standard one that I've seen others use before:  https://www.contributor-covenant.org/

File: .github/ISSUE_TEMPLATE/bug_website.yml
Comment (elreydetoda) [inline .github/ISSUE_TEMPLATE/bug_website.yml:30]: Is there a way to do an other option for this? There are some quite unique [browser options out there](https://destinationlinux.org/episode-269/). Most of them are all based on these, but some of them aren't.  So leaving that option open would probably be a good idea (and maybe an optional box below to say what browser if they chose other?), because I imagine it'll be those browsers we'll see the most bugs with 😅

File: .github/ISSUE_TEMPLATE/feature_request.yml
Comment (elreydetoda) [inline .github/ISSUE_TEMPLATE/feature_request.yml:20]: Maybe suggest to paste the link for where they’ve seen it before?

**Review by elreydetoda — APPROVED** [review]
A few small details/questions that I think might enhance it a bit more, but IMO optional. Very nicely done @CGBassPlayer !

It might be nice to have screenshots of each template as well, just so we can capture what they looked like for this PR.

---

Also, I don't known how familiar you are with `git`, but what I've done when I needed to merge something with my main branch to demonstrate functionality for something. Is after I merge it I reset it to JB repo's HEAD.

So, I've already added JB's repo as a remote with: `git remote add upstream https://github.com/JupiterBroadcasting/infra.git`

Then if I've merged my branch already and I want to reset my main branch I run: `git reset --hard upstream/main` and then I have to force push (since I'm re-writing history for my git repo): `git push -f`.

I will say though, be careful with force pushes (it can be a dangerous thing and unrecoverable (like how I messed up [here](https://github.com/JupiterBroadcasting/show-scraper/pull/28) and had to re-implement my previous work)), so just make sure to use it sparingly.

---

So besides the suggestions I've made and potential screenshots as enhancements everything from a functionality perspective LGTM. This is going to be an awesome addition to the repo! Thanks again @CGBassPlayer for doing all this research/work!

(Going to leave this open for @gerbrent since he showed interest in it before)

**Comment by CGBassPlayer** [comment]
Here are the screenshots so you can see what the forms look like. The table at the top is the meta data
`Name` and `Description` are what displays when you are choosing an issue type. And the `labels` are `assignees` are pretty straight forward. 

## Episode Issue
image

## Website Issue
image

## Development Issue
image

## Feature Request
image

I will look over (and probably implement) all your suggestions.

File: .github/ISSUE_TEMPLATE/bug_website.yml
Comment (CGBassPlayer) [reply .github/ISSUE_TEMPLATE/bug_website.yml:30]: I stole the dropdown idea from somewhere online. I just threw in the big 4 as a start. I am thinking I can add an `input` field for people to type in their browser if it is not listed.

File: .github/ISSUE_TEMPLATE/bug_shownotes.yml
Comment (CGBassPlayer) [reply .github/ISSUE_TEMPLATE/bug_shownotes.yml:47]: I have them commented out until we have something. That way it is ready to go when we are ready

**Review by elreydetoda — APPROVED** [review]
Awesome, thanks for implementing those changes @CGBassPlayer. LGTM :grin: 

(still going to let @gerbrent review it, but you've got my approval :partying_face: )

**Comment by gerbrent** [comment]
Love all of this!!! Super well thought out and implemented. .....launching!

**Comment by gerbrent** [comment]
woooooo!!!

image

**Pull request merged**
